documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -2155,13 +2139,6 @@ The following example sets the message key for the tables `inventory.customers`
  +
 In the preceding example, the columns `pk1` and `pk2` are specified as the message key for the table `inventory.customer`.
 For `purchaseorders` tables in any schema, the columns `pk3` and `pk4` serve as the message key.
-
-|[[db2-property-schema-name-adjustment-mode]]<<db2-property-schema-name-adjustment-mode,`+schema.name.adjustment.mode+`>>
-|avro
-|Specifies how schema names should be adjusted for compatibility with the message converter used by the connector. Possible settings:  +
-
-* `avro` replaces the characters that cannot be used in the Avro type name with underscore. +
-* `none` does not apply any adjustment. +
```

Comment:
I don't believe this should have been removed as this was added as a part of DBZ-3535 recently.

Reply:
I didn't remove this intentionally. My repo must be slightly out of synch and didn't pick up the change. Should I just re-add it and push the change?

Reply:
You could rebase your changes based on `main` as this should be part of `main` to get it.  If that doesn't work then yes please re-add.